content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -294,7 +295,7 @@ Server:  Message Received
 ### Procedure of How to run the two programs if using Intellij or any other IDE
```

Comment:
> ### Procedure of How to run the two programs if using Intellij or any other IDE
> 1. Compile the two 
> 2. First execute the program
> 3. On the server **run**, then run the program client **run* 
> 4. Type client window messages that are simultaneously received and shown by the Server window.
> 5. Type **BYE** to end.

There is a grammatical error in the third step but I do not understand what you are trying to say there so I want you to revise it. As well, the second **run** there is missing a `*` at the end (you put `**run*` instead of `**run**`). I would rephrase the fourth step to:

>Type into messages into the client's window. The server will also receive it so it should show up in the server's window.

for better grammar.